### PR TITLE
added ssh stage

### PR DESCRIPTION
**Pull request by gadiener**

### CODE REVIEW

File: ssh-quality.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+include:
+  - remote: 'https://raw.githubusercontent.com/jobtome-labs/ci-templates/v2.2.4/templates/ssh.yml'
+
+deploy:quality:image:
```
Comment (linuxbandit) [inline ssh-quality.yml:4]: name can be confused

File: ssh-production.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+include:
+  - remote: 'https://raw.githubusercontent.com/jobtome-labs/ci-templates/v2.2.4/ssh-quality.yml'
+
+deploy:quality:image:
```
Comment (linuxbandit) [inline ssh-production.yml:4]: name can be confused (even if they are in different files)

File: ssh-quality.yml (hunk 1 above)
Comment (gadiener) [reply ssh-quality.yml:4]: right

File: ssh-quality.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  extends: .ssh:exec
+  variables:
+    SSH_PRIVATE_KEY: ${SSH_PRIVATE_KEYT_QUALITY}
```
Comment (linuxbandit) [inline ssh-quality.yml:7]: oops typo we missed

**Pull request merged**